### DISCUSSION TITLE
[Bug]: Adding index does not have effect

### DISCUSSION THREAD

**Discussion by keatliang2005** [beyondcode/herd-community#89] [Q&A]
### Operating system version

macOS 13.4

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.1.1

### PHP Version

8.1.21

### Bug description

i have a site https://example.test, which default serve /index.php

but i do have /docs/index.html

i tried add 

in locker.test config file, tried to add index entries 
```nginx
server {
    listen 127.0.0.1:443 ssl http2;
    #listen 127.0.0.1:443 ssl http2; # valet loopback
    server_name locker.test www.locker.test *.locker.test;
    root /;
    ......
    
    location /docs {
        index index.html;
    }
    .......
}
```

save  and restart 

visit https://example.test/docs returns 404 
unless  https://example.test/docs/index.html

any help would be appreciated

**Comment by cord** [comment]
Works with custom LocalValetDriver https://github.com/beyondcode/herd-community/issues/122
